conf/nginx/sni_forward.conf

```diff
@@ -0,0 +1,27 @@
+{% set domain_ip_map = sni_forward_list.split(',') %}
+stream {
+
+    map $ssl_preread_server_name $name {
+        {% for domain_ip in domain_ip_map %}
+        {{ domain_ip.split(":")[0] }} {{ domain_ip.split(":")[0].replace('.', '') }};
```

Comment:
So a way to reduce the risk is to replace by `_` instead of nothing.

However it's not enough
```
domain_eu.org -> domain_eu_org 
domain.eu.org -> domain_eu_org
```

Reply:
I think you can't register a domain with a `_`.
Usually `_` are used in "special" subdomains (SRV records...)

Reply:
```suggestion
        {{ domain_ip.split(":")[0] }} {{ domain_ip.split(":")[0].replace('.', '_') }};
```